server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/model/assertion/NENAssertionTests.java

```diff
@@ -238,7 +241,7 @@ public void testNENEstimatedVaryingSamples(Integer auditedSamples, BigDecimal ri
    * there, the value matching the ID key is retrieved and the associated discrepancy type
    * incremented. If it is not there, then the discrepancy counts are not changed.
    */
-  @Test(expectedExceptions = {RuntimeException.class})
+  @Test
   public void testNENRecordNoMatch1(){
```

Comment:
These could be refactored into one big parameterized test.

Reply:
Same comment for NEBAssertionsTests.

Reply:
It would be a bit of a pain to do this, particularly as the expected outcomes/contexts are different enough for each case. However, there was an opportunity for abstracting out some common code across a subset of these record/remove tests (ie. checking of discrepancy counts).